packages/utils/src/db.ts

```diff
@@ -248,15 +262,17 @@ export class HoprDB {
     await this.put(UnAcknowledgedTickets(key), unacknowledged.serialize())
   }
 
-  async hasPacket(id: Uint8Array) {
-    if (await this.has(this.keyOf(PACKET_TAG_PREFIX, id))) {
-      await this.touch(this.keyOf(PACKET_TAG_PREFIX, id))
-      return true
+  async hasPacket(packetTag: Uint8Array) {
```

Comment:
this name is confusing as we are not just checking for a packet, we are setting it too. Please rename to make it clearer.

Reply:
changed, now obsolete